components/Toggle.jsx

```diff
@@ -13,25 +11,24 @@ const Toggle = () => {
   // ToggleButton is used to update the state variable `isAdmin`, which updates the icon displayed
   return (
     <div>
-      <ToggleButton
-        id="toggle-check"
-        type="checkbox"
-        variant="outline-primary"
-        checked={isAdmin}
-        onChange={() => {
-          setIsAdmin(!isAdmin);
-        }}
-      >
-        Status
-      </ToggleButton>
       {isAdmin ? (
-        <h6 className="flex float-right rounded-full border-solid border-2 border-black pb-1 px-1 bg-black text-white">
+        <h6
+          onClick={() => {
+            setIsAdmin(!isAdmin);
+          }}
+          className="flex float-right rounded-full border-solid border-2 border-black pb-1 px-1 bg-black text-white"
+        >
           admin.
-          <FaCircle />
+          <div className="bg-white rounded-full ml-2 mt-[3px] h-4 w-4"></div>
```

Comment:
use the FaCircle Icon and just change the text color

Reply:
Done ✔️